block/src/Pos/Block/Worker.hs

```diff
@@ -82,6 +82,8 @@ blkWorkers =
 
 informerWorker :: BlockWorkMode ctx m => (WorkerSpec m, OutSpecs)
 informerWorker =
+    -- TODO: [CSL-198] Catch and report non-critical errors, as
+    -- 'onNewSlot' no longer swallows all exception and rethrows them instead.
```

Comment:
I don't see why this worker may fail in normal cases, so let's stop program if it does (i. e. not catch anything here).